### ISSUE TITLE
Can

### ISSUE THREAD

**Issue by JackHenry1992** [GeeshangXu/mask-rcnn-pytorch#1]

**Comment by GeeshangXu** [comment]
yes, I am working on it pretraind model on COCO dataset will release soon.
